velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -175,5 +175,34 @@ TEST_F(DateTimeFunctionsTest, makeDate) {
   EXPECT_EQ(makeDate(2023, 3, 29), expectedDate);
 }
 
+TEST_F(DateTimeFunctionsTest, lastDay) {
+  const auto lastDay = [&](const std::optional<Date> date) {
+    return evaluateOnce<Date>("last_day(c0)", date);
+  };
+
+  auto checkEvaluation = [&](const auto& inputDate, const auto& expectedDate) {
+    Date d0;
+    Date d1;
+    parseTo(expectedDate, d0);
+    parseTo(inputDate, d1);
+    EXPECT_EQ(lastDay(d1), d0);
+  };
+  checkEvaluation("2015-02-28", "2015-02-28");
```

Comment:
It might be cleaner to write this as 

```
EXPECT_EQ(lastDay("2015-03-27"), "2015-03-31")
```

This way the error message would point out specific line in the test where the failure occurred.

Reply:
@mbasmanova done